plugins/batterystat.koplugin/main.lua

```diff
@@ -29,6 +27,7 @@ function State:toString()
 end
 
 local Usage = {}
+local INDENTATION = "   " --three spaces
```

Comment:
Been trying to unify comment style. :-)
```suggestion
local INDENTATION = "   " -- Three spaces.
```